app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt

```diff
@@ -211,15 +197,23 @@ class AppRatingDialogView(context: Context) : LinearLayout(context), OnRatingBar
 
     private fun setup(context: Context) {
         LayoutInflater.from(context).inflate(R.layout.component_app_rate_dialog, this, true)
-        ratingBar = findViewById(R.id.app_rate_dialog_rating_bar) as CustomRatingBar
-        commentEditText = findViewById(R.id.app_rate_dialog_comment_edit_text) as EditText
-        titleText = findViewById(R.id.app_rate_dialog_title_text) as TextView
-        descriptionText = findViewById(R.id.app_rate_dialog_description_text) as TextView
-        noteDescriptionText = findViewById(R.id.app_rate_dialog_note_description) as TextView
+        ratingBar = findViewById(R.id.app_rate_dialog_rating_bar)
+        commentEditText = findViewById(R.id.app_rate_dialog_comment_edit_text)
+        titleText = findViewById(R.id.app_rate_dialog_title_text)
+        descriptionText = findViewById(R.id.app_rate_dialog_description_text)
+        noteDescriptionText = findViewById(R.id.app_rate_dialog_note_description)
         ratingBar.setIsIndicator(false)
         ratingBar.setOnRatingBarChangeListener(this)
     }
 
     private val theme: Resources.Theme
         get() = context.theme
+
+    private fun getColorFromRes(@ColorRes colorResId: Int): Int {
+        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
You could use `ResourcesCompat.getColor(..)` instead. It does the same thing.

Reply:
right :)
corrected